### PR TITLE
Fallback subcategories

### PR DESCRIPTION
**Pull request by LuGary**
## Description

Adds the fallback subcategories for each parent category (excluding `OTHER`)

## Related Issues

- _[none]_

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Comment by LuGary** [comment]
Not sure why I can't request Jesse as a reviewer for this PR

**Comment by linh-transcend** [comment]
@LuGary Just to check, these fallback subcategories are the default subcategories for each category? Or I'm a misunderstanding

**Comment by LuGary** [comment]
@linh-transcend Yep that's right

File: src/subcategories/unspecified.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { makeEnum } from '@transcend-io/type-utils';
+
+export const UnspecifiedSubCategory = makeEnum({
+  /** Fallback subcategory */
+  Unspecified: 'unspecified',
```
Comment (anotherminh) [inline src/subcategories/unspecified.ts:5]: wait i thought we were going to name them "Other"

Comment (LuGary) [reply src/subcategories/unspecified.ts:5]: We will but that's a frontend thing, we need distinct names for the enum values

**Review by anotherminh — DISMISSED** [review]

**Review by anotherminh — APPROVED** [review]

**Pull request merged**